packages/flutter_tools/lib/src/base/os.dart

```diff
@@ -289,7 +290,18 @@ class _WindowsUtils extends OperatingSystemUtils {
   @override
   List<File> _which(String execName, { bool all = false }) {
     // `where` always returns all matches, not just the first one.
-    final ProcessResult result = _processManager.runSync(<String>['where', execName]);
+    ProcessResult result;
+    try {
+      result = _processManager.runSync(<String>['where', execName]);
+    } on ArgumentError {
```

Comment:
where you saying that `canRun` is not reliable on Windows?

Reply:
canRun just checks if the result of https://github.com/google/process.dart/blob/cce751da2076b8345d2f17b9db5cc6fc04f60e69/lib/src/interface/common.dart#L51 is null. I'm not actually sure what it will do in this case.